e2e_tests/tests/api_utils.py

```diff
@@ -390,3 +390,26 @@ def decorator(f: F) -> F:
         return f
 
     return decorator
+
+
+@functools.lru_cache(maxsize=1)
+def _get_streaming_updates_enabled() -> Optional[bool]:
+    config = _get_master_config()
+    if config is not None:
+        streaming_updates_enabled = "streaming_updates" in config["feature_switches"]
+        return streaming_updates_enabled
+    return None
```

Comment:
This is kind of counter-intuitive since usually feature switch is only true when specified (false if not-specified)

Reply:
Good catch! 

I think this pattern gives us `None` when there is an `APIException` or a `MasterNotFoundException`. So if we run the test, we’ll get an error, but if we skip it, no error will show up. I’m guessing that’s the intention behind this.

I will add a comment to make it clearer.